.github/workflows/test.yml

```diff
@@ -107,6 +107,24 @@ jobs:
       - name: Check Formatting
         run: sbt scalafmtCheckAll
 
+  mill:
```

Comment:
Could you please add this job to the dependencies of the `all_tests_passed` job?
Otherwise people will be able to ignore failing mill tests.